tools/CleanupBuild.ps1

```diff
@@ -47,11 +47,11 @@ foreach ($RMFolder in $resourceManagerFolders)
     Write-Verbose "Removing scripts and psd1 in $($RMFolder.FullName)"
     if (Test-Path -Path "$($RMFolder.FullName)\StartupScripts")
```

Comment:
Can you add the change here?

Reply:
@maddieclayton, I can add the fix here, but it is not required.  `Test-Path` and `Get-ChildItem` will properly resolve `\` and `/`.  The problem is the string comparison when `Get-ChildItem` returns "/a/cool/path", and we compare it to "/a/cool\path".

Reply:
Oh no worries then. LGTM